tests/test_address.py

```diff
@@ -152,7 +153,7 @@ def setup_method(self):
     def test_ip_resolver_with_hostname(self):
         with patch("dns.resolver.resolve") as resolve:
             answer = MagicMock()
-            answer.__str__.return_value = "48.225.1.104."
+            cast(MagicMock, answer.__str__).return_value = "48.225.1.104."
```

Comment:
I see no value gained from including the call to `cast` here. Is there a reason tooling is suggesting inclusion?

Reply:
Yes, pyright thinks `__str__` is a function, and doesn't recognize `return_value` as a valid param. I'm not sure why that is, might be a pyright bug